target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -85,6 +86,9 @@ ipq40xx_setup_macs()
 		wan_mac=$(mtd_get_mac_ascii devinfo hw_mac_addr)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		;;
+	mikrotik,lhgg-60ad)
+		lan_mac=$(mtd_get_mac_binary hard_config 16)
```

Comment:
Please use binary 0x10 for offset here.

Reply:
Will do, I have not updated it since all offsets have been migrated to hex instead of decimal